src/control/common/env_utils_test.go

```diff
@@ -126,21 +149,21 @@ func TestScrubProxyVariables(t *testing.T) {
 			disable:  "0",
 			expected: cleanEnv,
 		},
+		"disabled set to bananas": {
+			setup:    proxyEnv,
+			disable:  "bananas",
+			expected: cleanEnv,
+		},
```

Comment:
In the previous version, wasn't the behavior opposite? I.e. If the env variable was set to some weird value, it was taken as a true rather than a false. That sort of makes sense to me, as a C programmer anyway, that any value that isn't false is true.

Reply:
Nope, same behavior. If strconv.ParseBool() fails, I'd rather default to assuming that it wasn't set correctly. The alternative is to not bother parsing the value and just say that if it's set to anything it's assumed to be true, but that seems wrong from a safety/correctness perspective given that we can actually check the value and see if it resolves to a bool.